src/Sylius/Bundle/ApiBundle/Resources/config/validation/Customer.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
+
+    <class name="Sylius\Component\Core\Model\Customer">
+        <property name="gender">
+            <constraint name="Choice">
+                <option name="choices">
+                    <value>u</value>
```

Comment:
Should be possible to use consts over here

https://github.com/Sylius/Sylius/blob/0bc88ea1b9d4a03362b2373064a04ed2a395e982/src/Sylius/Component/Customer/Model/CustomerInterface.php#L21-L25

Reply:
I tried to do it, but it is not possible.